tests/util/fs.cpp

```diff
@@ -1,4 +1,6 @@
+#include <filesystem>
 #include <gtest/gtest.h>
+#include <iostream>
```

Comment:
is `iostream` used? and remove filesystem, use util/fs.h

Reply:
probably left over from testing